ci/appveyor/win64/python3.5.bat

```diff
@@ -0,0 +1 @@
+@C:\Python35-x64\python.exe %*

```

Comment:
I made ` ci/appveyor/win*/python*.bat` files, following the recommendation here:

https://tox.readthedocs.io/en/latest/developers.html#id2
https://github.com/tox-dev/tox/issues/114#issuecomment-349712050

Note that adding CI support for Python 3.6 and 3.7 requires adding more of those files.